TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/integ_test/integ_test/replayer_tests.py

```diff
@@ -0,0 +1,239 @@
+import pytest
+import unittest
+import requests
+import logging
+import secrets
+import string
+import time
+from http import HTTPStatus
+from requests import Session
+from requests.adapters import HTTPAdapter
+#from console_link.models.replayer import BaseReplayer, ECSReplayer
+from console_link.logic.clusters import connection_check, clear_indices, run_test_benchmarks, ConnectionResult
+from console_link.models.cluster import Cluster, AuthMethod
+from console_link.cli import Context
+from common_operations import (get_index, create_index, delete_index, get_document, create_document, delete_document,
+                               check_doc_match, check_doc_counts_match, generate_large_doc, execute_api_call,
+                               EXPECTED_BENCHMARK_DOCS)
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="class")
+def setup_replayer(request):
+    config_path = request.config.getoption("--config_file_path")
+    unique_id = request.config.getoption("--unique_id")
+    pytest.console_env = Context(config_path).env
+    pytest.unique_id = unique_id
+    source_cluster: Cluster = pytest.console_env.source_cluster
+    target_cluster: Cluster = pytest.console_env.target_cluster
+
+    # Confirm source and target connection
+    source_con_result: ConnectionResult = connection_check(source_cluster)
+    assert source_con_result.connection_established is True
+    target_con_result: ConnectionResult = connection_check(target_cluster)
+    assert target_con_result.connection_established is True
+
+    # Clear any existing non-system indices
+    clear_indices(source_cluster)
+    clear_indices(target_cluster)
+
+    #logger.info("Stopping replayer...")
+    #replayer: BaseReplayer = pytest.console_env.replayer
+    # TODO provide support for starting Replayer in Docker
+    #if replayer is not None and type(replayer) is ECSReplayer:
+    #    replayer.start()
```

Comment:
These tests are just interacting with the console, they won't be aware of docker vs ecs differences, no?